bower.json

```diff
@@ -11,7 +11,6 @@
     "spinner",
     "loading"
   ],
-  "private": true,
```

Comment:
Yes :)

All of our elements have `private: true`. That doesn't affect whether or not you can `bower install` the element, it just affects whether it's listed in the bower catalog. We prefer not to do that since a) we have the elements catalog, and b) a standalone element is kind of weird if you don't know what Polymer is, which you probably wouldn't if you're just browsing the bower catalog.